unittest/puppeteer/harness.js

```diff
@@ -22,7 +22,12 @@ const path = require('path');
 
 const {app} = require('../../app');
 
-const products = ['chrome', 'firefox'];
+const products = ['chrome'];
+if (process.platform !== 'win32' || Date.now() > Date.parse('2020-10-30')) {
```

Comment:
This will probably begin failing without the code changing. If just disabling on Windows with a TODO isn't palatable, then can you instead make this depend on the version of puppeteer, from `require('../../package.json').devDependencies.puppeteer`? I'd rather have it fail on upgrading Puppeteer than based on the calendar.

Also, to make it more visible that it's disabled, you might try `this.skip()` as described in https://mochajs.org/#inclusive-tests